source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+| Priority          | Other jobs with higher priority exist for the           |
+|                   | partition/reservation                                   |
++-------------------+---------------------------------------------------------+
+| Reservation       | The job is waiting for its reservation to become        |
+|                   | available                                               |
++-------------------+---------------------------------------------------------+
+| AssocMaxJobsLimit | The job is being held because the user/project has hit  |
+|                   | the limit on running jobs                               |
++-------------------+---------------------------------------------------------+
+| ReqNodeNotAvail   | The requested a particular node, but it's currently     |
+|                   | unavailable (it's in use, reserved, down, draining,     |
+|                   | etc.)                                                   |
++-------------------+---------------------------------------------------------+
+| JobLaunchFailure  | Job failed to launch (could due to system problems,     |
+|                   | invalid program name, etc.)                             |
++-------------------+---------------------------------------------------------+
+| NonZeroExitCode   | The job exited with some code other than 0              |
++-------------------+---------------------------------------------------------+
+
+Many other states and job reason codes exist. For a more complete description,
+see the :manpage:`squeue(1)` man page.
+
+
+Scheduling Policy
+-----------------
 
-.. code-block:: shell
+In a simple batch queue system, jobs run in a first-in, first-out (FIFO) order.
+This can lead to inefficient use of the system. If a large job is the next to
+run, a strict FIFO queue can cause nodes to sit idle while waiting for the
+large job to start. *Backfilling* would allow smaller, shorter jobs to use
+those resources that would otherwise remain idle until the large job starts.
+With the proper algorithm, they would do so without impacting the start time of
+the large job. While this does make more efficient use of the system, it
+encourages the submission of smaller jobs.
 
-  > module help gcp
 
-  ----------- Module Specific Help for 'gcp/2.2' --------------------
+Job priority
+------------
 
-  Sets up the shell environment for gcp
+Slurm on Gaea uses the `Slurm priority/multifactor plugin
+<https://slurm.schedmd.com/priority_multifactor.html>`_ to calculate a job's
+priority.  The factors used are:
 
+Age
+    the length of time ajob has been waiting in the queue, elgible to be
+    scheduled
 
-**module avail:** List all available modulefiles in the current
-MODULEPATH.
+Fair-share
+    the difference between the portion of the computing resource that has been
+    promised (allocation) and the amount of resources that has been consumed.
```

Comment:
either both "resources" or both "resource"

Reply:
fixed in. anew push.